.github/workflows/bdd.yml

```diff
@@ -0,0 +1,60 @@
+name: BDD Tests
+on:
+  push:
+    # branches:
+    #   - master
+    # paths-ignore:
+    # - 'README.md'
+    # - '.vscode/**'
+    # - '**.md'
```

Comment:
This should be uncommented. We need the tests to run in every push to master as well.